ownCloudAppShared/AppLock/AppLockManager.swift

```diff
@@ -417,7 +441,7 @@ public class AppLockManager: NSObject {
 	}
 
 	private var shouldDisplayCountdown : Bool {
-		if let startLockBeforeDate = self.lockedUntilDate {
+		if let startLockBeforeDate = self.lockedUntilDate, isLockBypassed {
```

Comment:
This condition _only_ shows the count down if the lock has been detected as having been bypassed - but not otherwise.